docs/integrations-guides/v1/dispersal/clients/eigenda-proxy-v1.md

```diff
@@ -0,0 +1,27 @@
+# EigenDA Proxy (V1)
+
+## About
+See [this page](../../../eigenda-proxy.md) for a high level description of the proxy.
+
+### Key Releases
+| version | (mainnet) max blob size | (holesky) max blob size | supported stacks   | Cert version(s) |
+|---------|-------------------------|-------------------------|--------------------|-----------------|
+|  v1.3.1 |        `2mib`           |          `2mib`         |      OP Stack      |      v0         |
+|  v1.4   |        `4mib`           |          `16mib`        |      OP Stack      |      v0         |
+|  v1.4.1 |        `16mib`          |          `16mib`        |  OP Stack, Orbit   |      v0         |
+
+## Recommended Config Types
+Different security measures and runtime optimizations can be applied through various proxy configurations. The different configuration flags can be found [here](https://github.com/Layr-Labs/eigenda-proxy?tab=readme-ov-file#configuration-options). The following recommendations are advised for different rollup node actor types:
```

Comment:
technically not true anymore since we removed the cfg flags from the readme. instead a prospective user would have to run `./bin/eigenda-proxy --help` to see flag options